app/controllers/orders_controller.rb

```diff
@@ -33,8 +34,26 @@ def index
                           .per(Settings.length.digit_8)
   end
 
+  def cancel
+    if @order.pending?
+      @order.update(status: :cancel)
```

Comment:
```suggestion
      @order.cancel!
```
đã nói là có enum hỗ trợ, mà cứ ko nghe

Reply:
chý ý là với method kết thúc với `!` thì phải làm j nữa nhé =))